.editorconfig

```diff
@@ -1,28 +1,33 @@
-root = true
```

Comment:
Why is this happening (I mean the diff shows that every line has changed). Is it because of the line endings?